src/components/ErrorNotifications/ErrorNotification.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+type Props = {
+  isErrorMessage: ErrorTypes,
+  setIsErrorMessage: (value: ErrorTypes) => void,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  isErrorMessage, setIsErrorMessage,
+}) => {
+  const [onCloseError, setOnCloseError] = useState(false);
+
+  const ErrorText = {
+    [ErrorTypes.load]: 'Unable to load a todo',
+    [ErrorTypes.upload]: 'Unable to add a todo',
+    [ErrorTypes.delete]: 'Unable to delete a todo',
+    [ErrorTypes.title]: 'Title can\'t be empty',
+    [ErrorTypes.update]: 'Unable to update a todo',
+    [ErrorTypes.none]: '',
+  };
```

Comment:
Move everything that can be moved out of component (if it has no deps on props, state etc.)